documentation/meta/incidents/runbooks/incident_response_runbook.md

```diff
@@ -0,0 +1,141 @@
+# Incident Response Runbook
+
+```{note}
+**Recording** an incident is a standalone task which can be performed by any maintainer. This recorder is not necessarily the one to stabilize the service or continue to work on incident triage.
+```
+
+```{warning}
+This runbook is intended to be used by Openverse maintainers, and references some resources that only Openverse staff have access to.
+```
+
+## 1. Record the incident
+
+When you observe a potential incident, you are responsible for recording it. Run
+the `/Incident Report` command in the maintainer’s `#openverse` Slack channel.
+This command will prompt for some basic information about the incident and then
+guide you into creating an incident report in the maintainer P2 blog.
+
+The command can also be run with a bookmarked "Report an Incident" link inside
+of Slack, which is found at the top of all `#openverse-` maintainer channels.
```

Comment:
Just a note for this, it'd be _awesome_ to move all of this communication over to the WordPress matrix at some point (with discussions in private channels as needed).